scout/server/blueprints/variants/forms.py

```diff
@@ -95,6 +95,7 @@ class VariantFiltersForm(FlaskForm):
     variant_type = HiddenField(default="clinical")
 
     gene_panels = NonValidatingSelectMultipleField(choices=[])
+    gene_panels_exclude = BooleanField("exclude panels")
```

Comment:
"exclude genes" instead of "exclude panels" perhaps?

Reply:
Mmm, definitely - and try to clarify on the interface that it will invert also the query on the Gene symbols row below! And capitalize, methinks.